fhir-server/src/test/java/com/ibm/fhir/server/util/FHIRRestHelperTest.java

```diff
@@ -1982,8 +1984,11 @@ public void testBundleSearchBundleWithNullRsrcAndNoId() throws Exception {
                     .build())
                 .build();
 
-        // Process bundle
-        Bundle responseBundle = helper.createSearchResponseBundle(Arrays.asList(null, patientNoId), context, "Patient");
+        // TODO make this work with the new search result semantics
```

Comment:
Does this TODO still need to be addressed?

Reply:
thanks, old TODO. Removed.